lib/diff.ml

```diff
@@ -17,11 +17,20 @@ type module_ = {
   mdiff : (module_declaration, module_modification) t;
 }
 
+and modtype = {
+  mtname : string;
+  mtdiff : (modtype_declaration, module_modification) t;
+}
+
 and module_modification = Unsupported | Supported of sig_item list
-and sig_item = Value of value | Module of module_
+and sig_item = Value of value | Module of module_ | Modtype of modtype
+
+type item_type = Value_item | Module_item | Modtype_item [@@deriving ord]
 
-type item_type = Value_item | Module_item [@@deriving ord]
-type sig_items = Val of value_description | Mod of module_declaration
+type sig_items =
+  | Val of value_description
+  | Mod of module_declaration
+  | Mtdtype of modtype_declaration
```

Comment:
What does `Mtdtype` stand for? I would have expected something close to `Mty` or `ModType`  as name for the variant...